### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by ruebot**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (ianmilligan1) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: Maybe specify "Desktop/Laptop" (don't laugh, I've seen this throw people off)

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```
Comment (ianmilligan1) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: add an `e.g.` to make clear that they should tweak these steps and not try to shoehorn their workflow into the ellipses

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (ianmilligan1) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: change example to 
`[e.g. MacOS 10.13.3]`

**Comment by SamFritz** [comment]
Looks great @ruebot! 

Just a quick question: what happens in the workflow if a user wants to request a feature (e.g. for AUK) but doesn't have GitHub account? I know we want to encourage everyone to fill out the template through Git, but just thinking about someone who may only be using AUK through the GUI. 

(this may be a questions for down the line).

**Comment by ianmilligan1** [comment]
Good point! If they click "new issue" w/o an account they'll get this.

screen shot 2018-05-02 at 3 16 54 pm

So maybe in places on the platform, etc. where we provide a link to "open an issue," we could also have an invitation to join our Slack group or e-mail us?

If that sounds good, we can merge this and keep this in mind for places where we invite people to open issues?

**Pull request merged**